litmus-portal/k8s-manifest.yml

```diff
@@ -161,7 +154,7 @@ spec:
     - ReadWriteOnce
   resources:
     requests:
-      storage: 20Gi
+      storage: 4Gi
```

Comment:
Why was this changed? It is too low for production

Reply:
Yeah removing it, just for okteto this was needed.